enhancements/agent-installer/simplified-registry-less-cluster-operations.md

```diff
@@ -0,0 +1,522 @@
+---
+title: simplified-registry-less-cluster-operations
+authors:
+  - "@andfasano"
+reviewers:
+  - "@danielerez"
+  - "@oourfali"
+  - "@bfournie"
+  - "@rwsu"
+  - "@pawanpinjarkar"
+  - "@yuqi-zhang"
+  - "@mrunalp"
+  - "@sdodson"
+approvers:
+  - "@zaneb"
+api-approvers:
+  - "@joelspeed"
+creation-date: 2025-07-07
+last-updated: 2025-07-07
+tracking-link: 
+  - "https://issues.redhat.com/browse/AGENT-1250"
+  - "https://issues.redhat.com/browse/VIRTSTRAT-60"
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Simplified registry-less cluster operations for disconnected environments
+
+## Summary
+
+This enhancement proposal outlines an opinionated workflow to simplify
+on-premises cluster operations - such as installation, upgrade and node
+expansion - by eliminating the need to setup and configure an external
+registry mirror. This approach is specifically designed for disconnected
+environments and leverages existing tools such as Agent-based Installer,
+Assisted Installer and OpenShift Appliance.
```

Comment:
I think I understand that this approach is simpler because it doesn't involve users setting up mirror registries on their disconnected network, so are we planning to funnel _all_ disconnected users to this approach?

If not, why would a customer need to stick with the traditional oc-mirror approach?
Or if so, should oc-mirror be able to directly create the ISO (and associated cluster objects) to avoid fragmentation of tooling, docs, etc.?